lib/server/utilities/index.js

```diff
@@ -13,8 +13,10 @@
 
 const appendPath = require('./appendPath');
 const dtoValidator = require('./dtoValidator');
+const { findAndCountAllToHttpView } = require('./findAndCountAllToHttpView.js');
 
 module.exports = {
     appendPath,
     dtoValidator,
+    findAndCountAllToHttpView,
```

Comment:
I see no point in not using the function directly

Reply:
yes refactored, but ... as it's not belong to scope of this PR I will add in another one